### PR TITLE
feat(cf-field): add optional hints for form fields

### PR DESCRIPTION
**Pull request by luytena**
Add option to create hints for certain question types. These
are displayed below the input field and can be used to provide
short, informative messages. Hints are available for all question
types except for form, static and action button questions.

image

### CODE REVIEW

**Comment by anehx** [comment]
How about a question mark icon before the hint? Also, do I see correctly that the error and the hint have different font sizes?

**Comment by luytena** [comment]
> How about a question mark icon before the hint? Also, do I see correctly that the error and the hint have different font sizes?

You are right, I fixed the font sizes. About the question mark icon, we were thinking that it might take up too much focus. This is how it would look like:

image

**Comment by anehx** [comment]
Fair enough!

**Review by czosel — APPROVED** [review]
LGTM :+1:

**Comment by anehx** [comment]
@luytena please rebase, fix the caluma container version to the latest version (not dev) and run `yarn update-caluma`. After that we can merge!

File: docker-compose.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 version: "3"
 services:
   db:
-    image: postgres:latest
+    image: postgres:alpine
```
Comment (anehx) [inline docker-compose.yaml:4]:
```suggestion
    image: postgres:14-alpine
```

See https://github.com/projectcaluma/caluma/pull/1704/commits/29614e901c51c9a73d417a3174507b2cec62049d

**Comment by luytena** [comment]
There were a number of merge conflicts, mostly in the `package.json` files. I resolved them by selecting the most recent version.

**Comment by anehx** [comment]
@luytena ember-uikit 5.0.0 is more recent than 5.0.0-beta.4 :wink:

**Comment by luytena** [comment]
Should be correctly rebased now.

**Review by anehx — APPROVED** [review]

**Comment by anehx** [comment]
Awesome work, looking forward to using this! :100:

**Comment by luytena** [comment]
Thanks for fixing the tests!

**Pull request merged**